middleware.js

```diff
@@ -4,8 +4,7 @@ import App from './app/app'
 
 module.exports = (options) =>
   (req, res, next) => {
-    const endpointRegExp = new RegExp(options.middlewarePath, 'i')
-    if (endpointRegExp.test(req.url)) {
+    if (req.url.match(new RegExp(options.middlewarePath, 'i'))) {
```

Comment:
I agree with [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match): If you need to know if a string matches a regular expression RegExp, use search() (or test()), not match()